app/components/footer_component.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class FooterComponent < ViewComponent::Base
+  delegate :scotland?, :scotland_public_website_footer_nav_links, :public_website_footer_nav_links, to: :helpers
+
+  def initialize(current_path:, feedback_survey_id:, columns:, feedback_title: nil, legal_summary: nil)
+    super
+    @current_path = current_path
+    @feedback_survey_id = feedback_survey_id
+    @feedback_title = feedback_title
+    @columns = columns
+    @legal_summary = legal_summary
```

Comment:
`legal_summary` isn't set in the layout - so it's always going to be nil.  If that's correct, we can get rid of it 🚮